docs/python_binding.rst

```diff
@@ -14,37 +14,39 @@ building
 Building the libtorrent python bindings will produce a shared library (DLL)
 which is a python module that can be imported in a python program.
 
-building using setup.py
------------------------
+building using boost build (windows)
+------------------------------------
 
-There is a ``setup.py`` shipped with libtorrent that can be used on windows.
-On windows the setup.py will invoke ``bjam`` and assume that you have boost
-sources at ``$BOOST_PATH``. The resulting executable is self-contained, it does
-not depend any boost or libtorrent dlls.
+Download and install `Visual C++ 2015 Build Tools <http://landinghub.visualstudio.com/visual-cpp-build-tools/>`_
 
-On other systems, the setup.py is generated by running
-``./configure --enable-python-binding``.
+Download `Boost libraries <http://www.boost.org/users/history/>`_ Extract it to c:\\Libraries\\boost_1_63_0 and create these environmental vars:
 
-To build the Python bindings do:
+1. BOOST_BUILD_PATH: "c:\\Libraries\\boost_1_63_0\\tools\\build\\"
+2. BOOST_ROOT: "c:\\Libraries\\boost_1_63_0\\"
 
-1. Run::
+Navigate to BOOST_ROOT, execute "bootstrap.bat" and add to the path "c:\\Libraries\\boost_1_63_0\\tools\\build\\src\\engine\\bin.ntx86\\"
+	
+Move the file ``$user-config.jam`` from ``$BOOST_BUILD_PATH\example\`` to ``$BOOST_BUILD_PAT\\user-config.jam`` and add this at the end:
+1. using msvc : 14.0 ;
+2. using python : 3.5 : C:/Users/<UserName>/AppData/Local/Programs/Python/Python35 : C:/Users/<UserName>/AppData/Local/Programs/Python/Python35/include : C:/Users/<UserName>/AppData/Local/Programs/Python/Python35/libs ;
+(change the python path for yours)
 
-	python setup.py build
-
-2. As root, run::
-
-	python setup.py install
-
-
-building using boost build
---------------------------
+Navigate to binding\\torrents and execute::
+	python setup.py build --bjam
+	
+Note: If you are using 64bits python you should edit setup.py and add this to the b2 command:
+``$address-model=64``
 
+This will create the file libtorrent.pyd inside build\\lib\\ that contains the binding.
+	
+building using boost build (others)
+-----------------------------------
 To set up your build environment, you need to add some settings to your
 ``$BOOST_BUILD_PATH/user-config.jam``.
 
 Make sure your user config contains the following line::
 
-	using python : 2.3 ;
+	#using python : 2.3 ;
```

Comment:
yes, but when you open the file for the first time this line is commented. I explain in the next lines that you should change this to your python version

Reply:
I think it's risky to document what the default ``user-config.jam`` file will look like, since it's likely to change in a future version of boost. It may also be somewhat misleading, for a reader skimming, it may not be obvious that what you're showing is what *not* to have in the file. I think it would be better to just say what *should* be in the file.